src/wallet/wallet2.cpp

```diff
@@ -115,6 +115,8 @@ using namespace cryptonote;
 #define STAGENET_SEGREGATION_FORK_HEIGHT 1000000
 #define SEGREGATION_FORK_VICINITY 1500 /* blocks */
 
+static const std::string PUB_KEY_SIGNATURE_MAGIC = "SigPkV1";
```

Comment:
How about making this have multisig in the name (or some shortening thereof) for clarity ?

Reply:
I thought about it, but verification function accepts public key so it potentially maybe a common function and i didn't want to bind it to mulitisig only

Reply:
Or maybe you mean variable name (not the magic itself)? If so probably it worth to do it...

Reply:
I meant the magic. This is multisig specific, and a multisig wallet may be able to sign with proper threshold in the future.

Reply:
Okay, will do it in a day